### ISSUE TITLE
(Need to complete this during the week of 7/18) Documentation support for an accessibility review of the Q2 working demo model form

### ISSUE THREAD

**Issue by taylorkaren** [department-of-veterans-affairs/va-forms-system-core#464]
## Context
- We want a first pass review on the accessibility status of the Q2 working demo model form so that we can assess how well we fulfilled our "accessible by default" goal and what specific or systemic accessibility problems need to be addressed before moving forward with additional work on the new VA Forms System Core (VAFSC).
- A couple of things we know for sure need to be checked:
    - We should be able to do some basic accessibility checks on the code side to confirm correct implementation of things like labels on all buttons, links and headers for screen reader accessibility.
    - Ensure meaningful error feedback is provided.
- What is the experience like when using assistive technology with the working demo model form?

## Task Description
To dos from the 7/13/22 planning meeting:
- [x] Document what parts of the demo form come from the Design System and what is VAFSC.
- [x] Document the demo form conditional elements - what is conditionally displayed on a page and what full pages are conditionally displayed, along with the conditions that trigger the display. Screen shots will be helpful for this, we can also reference the UX Pin Prototypes ([Desktop](https://preview.uxpin.com/5b0d283a5e66f0f1aea71cf18f2e50e6e142f362#/pages/150714454/simulate/sitemap)) or [Mobile](https://preview.uxpin.com/62a23fc3df98ec0e6916257a256070f85961318a#/pages/150534574/simulate/sitemap)).

## Acceptance Criteria
- [x] Confluence page has been created in the [Accessibility Documentation section of the FLT Confluence space](https://vfs.atlassian.net/wiki/spaces/FLT/pages/2269184105/Accessibility+Documentation) that documents what parts of the demo form come from the Design System and what is VAFSC.
- [x] Confluence page has been created in the [Accessibility Documentation section of the FLT Confluence space](https://vfs.atlassian.net/wiki/spaces/FLT/pages/2269184105/Accessibility+Documentation) that documents demo form conditional elements - what is conditionally displayed on a page and what full pages are conditionally displayed, along with the conditions that trigger the display.
- [x] Karen has sent links to these pages out to the group that met in the 7/13/22 accessibility audit planning meeting.

**Comment by obliviga** [comment]
Hi @taylorkaren. The two documents have been created and the links for them can be accessed here: https://vfs.atlassian.net/wiki/spaces/FLT/pages/2269184105/Forms+System+Core+Accessibility+Documentation

**Comment by taylorkaren** [comment]
@obliviga: I have reviewed the two pages. Component Sources looks good, I had quite a few comments on Conditional Elements and also did some formatting updates on that page.

Let's talk in standup today about how we can find answers to your "based on what?" questions.

**Comment by obliviga** [comment]
@taylorkaren, thank you for your feedback. I've applied the feedback and cleaned up the page. It's ready for review again.

**Comment by taylorkaren** [comment]
@obliviga: Reviewed the page again, two comments to address and then we should be done.

**Comment by obliviga** [comment]
@taylorkaren, thank you for that feedback! I've updated the document.
